core/src/main/scala/stryker4s/mutants/findmutants/MutantMatcher.scala

```diff
@@ -37,6 +38,10 @@ class MutantMatcher()(implicit config: Config) {
     case Or(orig)  => orig ~~> And
   }
 
+  def matchConditionalExpression: PartialFunction[Tree, Seq[Option[Mutant]]] = {
+    case IfStatementCondition(orig, mutatedTerms)  => orig ~~> (mutatedTerms: _*)
```

Comment:
See my above comment, it makes more sense to stay in line where this class is responsible for matching statements to mutators, and the mutationType for finding a node in a tree that can be mutated